### ISSUE TITLE
Obsolete column in SQL table usrData

### ISSUE THREAD

**Issue by vmario89** [nuxsmin/sysPass#846]
Hello,

i found a table column which is not referenced from sysPass code. Seems to be obsolete:
![grafik](https://user-images.githubusercontent.com/5631071/35184502-e63d9dec-fdf6-11e7-86d1-9fbdefd9c673.png)

regards, Mario

**Comment by nuxsmin** [comment]
Right! It will be removed in next release. It's there because I thought on implementing the secondary groups in the users table instead of the groups one.

Thanks for the feedback!

Regards
